pkg/component/registrycaches/registry_caches.go

```diff
@@ -236,54 +282,46 @@ func (r *registryCaches) computeResourcesDataForRegistryCache(ctx context.Contex
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name + "-config",
 			Namespace: metav1.NamespaceSystem,
-			Labels:    getLabels(name, upstreamLabel),
+			Labels:    registryutils.GetLabels(name, upstreamLabel),
 		},
 		Data: map[string][]byte{
 			"config.yml": configYAML.Bytes(),
 		},
 	}
 	utilruntime.Must(kubernetesutils.MakeUnique(configSecret))
 
-	service := &corev1.Service{
+	secretTLS := &corev1.Secret{
```

Comment:
```suggestion
	tlsSecret := &corev1.Secret{
```

to keep it identical with the existing `configSecret` naming.